orchestrator/tests/test_plugin_runtime_integration.py

```
@@ -0,0 +1,186 @@
+
+    @pytest.mark.asyncio
+    @patch("core.services.plugin_cache.get_plugin_content_cache")
+    async def test_tier2_loaded_for_relevant_plugins(self, mock_get_cache, mock_db):
+        """When task context matches a plugin, its SKILL.md content appears."""
+        rows = [
+            _make_plugin("email-sender", "Email Sender", "Send emails", ["email"], priority=0),
+            _make_plugin("db-tool", "DB Tool", "Query databases", ["database"], priority=1),
+        ]
+
+        cache = MagicMock()
+        cache.get_file = AsyncMock(side_effect=lambda slug, ver, path: {
```

Ruff (0.14.14)

[warning] Unused lambda argument: `ver`

(ARG005)